.github/tests/javascript/graphql-sdl-first/run.sh

```diff
@@ -2,13 +2,14 @@
 
 set -eu
 
-yarn 
-yarn prisma db push --preview-feature
-yarn dev &
+npm install
+npx prisma migrate dev --name init
+npx prisma db seed --preview-feature
+npm run dev &
 pid=$!
 
-sleep 10
+sleep 20
 
-curl --fail 'http://localhost:4000/' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'Connection: keep-alive' -H 'DNT: 1' -H 'Origin: http://localhost:4000' --data-binary '{"query":"query {\n  feed {\n    id\n    content\n    author {\n      id\n      name\n      email\n    }\n  }\n}"}' --compressed
+npx newman run https://raw.githubusercontent.com/prisma/prisma-examples/latest/.github/tests/postman_collections/graphql.json --bail
```

Comment:
Would it possible make sense to keep both the newman tests and the curl sanity check that was there before?

Reply:
The curl sanity check wouldn't be as helpful when working with GraphQL as it always returns an HTTP 200, except when there's a server failure. 

The tests from the postman collection validate the status code and response body of every request.